source/images/opened-eye.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 764.3 400">
```

Comment:
Remake this, but not cropping the image area to this content, but to the crossed eye content, so they will be aligned to each other.